tests/unit/test_database.py

```diff
@@ -15,19 +15,35 @@
 
 import database_observer
 import synapse
+from charm import SynapseCharm
 from charm_types import DatasourcePostgreSQL
-from constants import SYNAPSE_CONTAINER_NAME
+from constants import SYNAPSE_CONTAINER_NAME, TEST_SERVER_NAME
 from database_client import DatabaseClient
 from exceptions import CharmDatabaseRelationNotFoundError
 
 
-def test_erase_database(harness_with_postgresql: Harness, monkeypatch: pytest.MonkeyPatch) -> None:
+def test_erase_database(
+    datasource_postgresql_password: str, monkeypatch: pytest.MonkeyPatch
+) -> None:
     """
     arrange: start the Synapse charm, set Synapse container to be ready and set server_name.
     act: add database relation and erase database.
     assert: erase query is executed.
     """
-    harness = harness_with_postgresql
+    harness = Harness(SynapseCharm)
+    harness.update_config({"server_name": TEST_SERVER_NAME})
+    relation_id = harness.add_relation("database", "postgresql")
```

Comment:
Iirc there's a way now to do this in a single line

Reply:
Thanks, I have made the changes here : https://github.com/canonical/synapse-operator/pull/36/commits/c331ecb1a5a485378059e08ebc76026e9c7e7c78